.github/workflows/scan-images.yaml

```diff
@@ -0,0 +1,49 @@
+name: Scan images
+on:
+  workflow_dispatch:
+    inputs:
+      bundle-file:
+        description: Location of bundle file for the release
+        default: ''
+        required: true
+        type: string
+
+jobs:
+  scan:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Scan
+        run: |
+          sudo snap install yq
+          CI_REPO="https://github.com/canonical/kubeflow-ci.git"
+          mkdir -p kubeflow-ci
+          cd kubeflow-ci
+          git init -q
+          git remote add -f origin "$CI_REPO" &> /dev/null
+          git sparse-checkout set scripts/images/
+          # using test branch
+          git pull -q origin main
```

Comment:
This can be replaced with the checkout action.  I think something like:

```
- uses: actions/checkout@v3
  with: 
    repository: https://github.com/canonical/kubeflow-ci.git
    sparse-checkout: scripts/images/
````

Reply:
+1